src/java/org/apache/cassandra/db/commitlog/CommitLogReplayer.java

```diff
@@ -18,30 +18,59 @@
  */
 package org.apache.cassandra.db.commitlog;
 
-import java.io.*;
-import java.util.*;
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.EOFException;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
 import java.util.concurrent.Future;
 import java.util.concurrent.atomic.AtomicInteger;
 
-import com.google.common.base.Predicate;
```

Comment:
The google imports and maybe others are being moved to where they aren't supposed to be according to http://wiki.apache.org/cassandra/CodeStyle